dss/__init__.py

```diff
@@ -16,6 +16,8 @@
 # Constants common to the indexer and query route.
 DSS_ELASTICSEARCH_INDEX_NAME = "hca"
 DSS_ELASTICSEARCH_DOC_TYPE = "hca"
+DSS_ELASTICSEARCH_QUERIES_INDEX_NAME = "queries"
```

Comment:
For the index that will contain the subscriptions, I prefer the name "subscriptions" rather than "queries" (both in the constant name and value).